README.md

```diff
@@ -1,2 +1,38 @@
-A sample command-line application with an entrypoint in `bin/`, library code
-in `lib/`, and example unit test in `test/`.
+# [Evoleen](https://www.evoleen-technology.com) Fhir Rest Client
+
+This package helps developers to retrieve and manipulate FHIR data with REST client.
+
+This package does not manage authentication into a fhir server, for this feature exists [Azure Identity](https://pub.dev/packages/azure_identity)
```

Comment:
would remove this sentence - Azure Identity is a valid system to use, but could also use no authentication, OAuth, etc